src/vm/eetwain.cpp

```diff
@@ -4816,8 +4816,22 @@ bool EECodeManager::EnumGcRefs( PREGDISPLAY     pContext,
                         printf("%cP+%02XH]: ", frameType, +dspOffs);
                 }
 #endif
+
+                unsigned flags = CHECK_APP_DOMAIN;
+#ifndef WIN64EXCEPTIONS
```

Comment:
Do you need to change the code above that references this_OFFSET_FLAG? (since that is no longer used?)